### DISCUSSION TITLE
Can't get 2 or more columns out of DataFrame

### DISCUSSION THREAD

**Discussion by levyitay** [opensource9ja/danfojs#218] [General] [answered]
When trying to get 2 or more columns out of the DataFrame, only getting the last column

**To Reproduce**
```js
const df = new dfd.DataFrame([{a:1,b:2,c:3},{a:11,b:22,c:33},{a:111,b:222,c:333}]
df['a','b'].print()
```

**Expected behavior**
```js
df['a','b'] 
```
should return a,b columns

**Comment by risenW** [comment, marked as answer]
@levyitay If you need to access more than one column, use the loc function as demonstrated here: https://danfo.jsdata.org/api-reference/dataframe/danfo.dataframe.loc

**Reply by levyitay** [reply]
Thanks, missed that
